extensions/wikia/UserActivity/styles/UserActivity.scss

```diff
@@ -0,0 +1,118 @@
+@import "skins/shared/mixins/arrow";
+@import "skins/shared/mixins/clearfix";
+@import "skins/shared/color";
+
+.no-image {
+  background-color: #f0f0f0;
+  background-image: url(/extensions/wikia/VideoHandlers/images/Brackets.png);	/* inline */
+  background-position: 50px 25px;
+  background-repeat: no-repeat;
+  height: 90px;
+  width: 160px;
+}
+
+/* The Grid ---------------------- */
+.row { width: 940px; max-width: 100%; min-width: 768px; margin: 0 auto; }
```

Comment:
Inline rules are against ["Only one rule per line" in the coding conventions](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#whitespace-and-structure).